lib/frontend/github-client-middleware.js

```diff
@@ -1,21 +1,18 @@
-const octokit = require('@octokit/rest');
+const { Octokit } = require('@octokit/rest');
 
 module.exports = (getAppToken) => (req, res, next) => {
-  res.locals.github = octokit();
-
   if (req.session.githubToken) {
-    res.locals.github.authenticate({
-      type: 'token',
-      token: req.session.githubToken,
+    res.locals.github = new Octokit({
+      auth: req.session.githubToken,
     });
+  } else {
+    res.locals.github = new Octokit();
```

Comment:
this is an unauthenticated client, which was the case before if `req.session.githubToken` didn't exist.